core-kotlin-modules/core-kotlin-collections-list-2/src/test/kotlin/listOfLists/ListOfListsUnitTest.kt

```diff
@@ -0,0 +1,53 @@
+package listOfLists
+
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.Test
+import kotlin.test.assertTrue
+
+class ListOfListsUnitTest {
+
+    @Test
+    fun `test creating a list of lists using listOf()`() {
```

Comment:
Can we rename the tests to follow the proper naming convention? You can find it here: https://docs.google.com/document/d/1I6eu-eH5D1E0Mkl0Al-4xTgYiK2Ny0PPGtXaN2ToepI/edit

Example: 
test creating a list of lists using listOf() -> Creates a list of lists when using listOf()